.github/CODEOWNERS

```
@@ -224,6 +225,7 @@
 /comp/core/sysprobeconfig @DataDog/ebpf-platform
 /comp/core/workloadmeta @DataDog/container-integrations
 /comp/dogstatsd/statsd @DataDog/agent-shared-components
+/comp/forwarder/orchestrator @DataDog/agent-metrics-logs
```

```suggestion
/comp/forwarder/orchestrator @DataDog/agent-metrics-logs @DataDog/container-app
```
We should probably own this as well as it's very specific to our product